.circleci/config.yml

```diff
@@ -244,13 +188,14 @@ jobs:
       - upload_artifacts
 
   server-wallet-stress-test-threaded:
-    <<: *defaults
+    <<: *postgres
     resource_class: xlarge
     environment:
-      PSQL_ARGS: -p 5432 -h localhost
       NODE_ENV: test
+      PSQL_ARGS: -p 5432 -h localhost -U postgres
+      SERVER_DB_USER: postgres
+      POSTGRES_USER: postgres     
```

Comment:
Aren't these in `postgres`?

Reply:
Yes they are. But I believe the whole `environment` key will be overwritten, so it is necessary to specify them again.